hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/InfoSubcommand.java

```diff
@@ -105,10 +108,20 @@ public void execute(ScmClient scmClient) throws IOException {
           && spec.root().userObject() instanceof GenericParentCommand
           && ((GenericParentCommand) spec.root().userObject()).isVerbose();
       if (verbose) {
-        LOG.info("Pipeline Info: {}", container.getPipeline());
+        Pipeline pipeline = null;
```

Comment:
We can use the same pipeline info object as above (container.getPipeline()) as we have introduced a new field to denote original write pipeline.